src/containers/App.jsx

```diff
@@ -55,17 +55,21 @@ const App = ({ accounts, konnectors, triggers }) => {
 
   useEffect(() => {
     setIsFetching(
-      [accounts, konnectors, triggers].some(collection =>
+      [accounts, konnectors].some(collection =>
         ['pending', 'loading'].includes(collection.fetchStatus)
       )
     )
     setHasError(
-      [accounts, konnectors, triggers].find(
+      [accounts, konnectors].find(
         collection => collection.fetchStatus === 'failed'
       )
     )
-  }, [accounts, konnectors, triggers])
-
+  }, [accounts, konnectors])
+  useEffect(() => {
+    client.query(Q('io.cozy.triggers'))
+    client.query(Q('io.cozy.jobs'))
+    client.query(Q('io.cozy.accounts'))
+  }, [])
```

Comment:
The useEffect with no dep will be called at every render, right? Is it what we want?

Reply:
I'm not sure what we want to do here with the data, but those queries will fetch the 100 first triggers / jobs / accounts. Is it expected?

Reply:
> The useEffect with no dep will be called at every render, right? Is it what we want?

No, [], means only at the first render. It's the same as componentDidMount()

Reply:
> I'm not sure what we want to do here with the data, but those queries will fetch the 100 first triggers / jobs / accounts. Is it expected?

This is already the case I think, I will not change that right now.